v3/integration/config.json

```diff
@@ -755,12 +755,8 @@
     "w_subject_dn_trailing_whitespace": {
       "WarnCount": 64
     },
-    "w_tls_server_cert_valid_time_longer_than_397_days": {},
-    "e_cert_sig_alg_not_match_tbs_sig_alg": {
-      "ErrCount": 10
```

Comment:
...and on this one I spent another good chunk of time trying to figure out why both having this and removing it still results in a pass in the corpus.

...

It's because it's a duplicate of the same entry at L296.

![image](https://user-images.githubusercontent.com/8518514/123523146-ba98e780-d676-11eb-8eab-7bdb5d161694.png)